tmt-app/src/Components/Home.js

```diff
@@ -103,19 +90,20 @@ function Home(){
                         </Typography>
                     </PrimaryBtn>
  
-                    <div id="Home_Button_Main_Link">
-                        <Link
-                            sx={{color:"Boba.main"}}
-                            underline="none"
-                            href="google.com" 
-                            target="_blank"
-                            rel="noopener noreferrer" 
-                            aria-label="Learn more about other different majors"
-                        >
-                            <Typography variant="CustomSubHeading">
-                                See all Majors/Careers
-                            </Typography>
-                        </Link>
+                    <div className="Home_Button_Main_Link">
+                        <RouterLink to="/guide" >
+                            <Link
```

Comment:
https://mui.com/material-ui/guides/composition/#inlining-amp-caveat

Instead of stacking components like this, MUi has a `component` prop as part of their composition api we can leverage.